### PR TITLE
Add Nav component and styling

### PR DESCRIPTION
**Pull request by apete12**
What I did:

- Add Nav component and associated styling. Feature does not add or change functionality. 

Closes issue #16 

Did this break anything?

- [X] No
- [ ]  Yes

Type of change

- [X] New feature
- [X]  Styling 
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:

- [ ]  If this code needs to be tested, all tests are passing
- [X]  The code runs locally
- [ ]  There are comments where clarification/ organization is needed
- [X]  The code is DRY. If it's not, I tried my best
- [X]  I reviewed my code before pushing

What's next?
- Refactor App component to include network request to individual movie data
- Incorporate React Router
- Implement Cypress e2e testing

### CODE REVIEW

**Review by JoshBennett793 — CHANGES_REQUESTED** [review]
After reviewing these changes on my local machine, the header does appear to have a margin around it or a parent container has a padding attribute causing there to be space between the  header and the sides of the viewport. Could we resolve that and close that gap please?

Perhaps a better name for this feature would be a header? Let me know what you think.

File: src/components/Nav/Nav.jsx
Comment (JoshBennett793) [inline src/components/Nav/Nav.jsx:1]: If this were a header, we could use the more semantic header tag here

Comment (apete12) [reply src/components/Nav/Nav.jsx:1]: Nice catch on the gap! Will change that. 

In terms of the component name, I chose Nav with our discussion of implementing a search bar in mind. Just did some research (this [article](https://dev.to/masakudamatsu/don-t-nest-nav-inside-header-do-nest-the-hamburger-menu-button-inside-nav-6cp) was informative) and it seems like header is indeed the best option, then if we decide to include the search bar `<form role="search"> `(the search role) is appropriate for accessibility.

Comment (JoshBennett793) [reply src/components/Nav/Nav.jsx:1]: Interesting! We could nest the header inside of a div for now and when it comes time to implement the search bar, we have the div ready to hold our search bar. I love the accessibility features in the article's navigation drawer. I'm saving that for later.

**Review by JoshBennett793 — APPROVED** [review]

**Comment by JoshBennett793** [comment]
Awesome!

**Pull request merged**